texmf/tex/latex/fks/fksmeta.sty

```diff
@@ -81,9 +81,35 @@
 \met@probheader & %
 \Roman{solvedbatch} & \textnormal{\%} & $\Sigma$ \\}
 
+% For switching between FYKOS and Vyfuk functionality across texmf files.
```

Comment:
@Werkov: 

Používají se tyhle ify někde?

Chtěl bych vystačit jen s optionami, potažmo meta-makry.

@LukasLedvina 

Souhlasím, také by se mi to více líbilo.

Ano, používají. Resp. nevím jak jinak napsat:
```
if(fykos) {...}
if(fykos && czech) {...}
if(fykos && english) {...}
if(vyfuk) {...}
```

Reply:
Našel jsem jen použití iffkslang v fksmeta.sty:104 a moc to použití nechápu.

Reply:
Já to chápu tak, že by se to v uvozovkách dalo nahradit if (not czech && not slovak && not english) bez nutnosti zavádět další if. Ale -- nevím, zda jste si toho všimli, používat složitější konstrukce s \newif je pain.

Tady: https://tex.stackexchange.com/questions/5894/latex-conditional-expression navrhuje používat např. etoolbox. Jinak, tady https://tex.stackexchange.com/questions/2936/test-whether-token-list-is-empty/2951#2951 vysvětlují, proč je to pain (nedá se to jednoduše uzávorkovat a špatně se matchují else větve). Takže možná bych to přepsal pomocí toho etoolboxu (ideálně všechny ify). Zase otázka, možná by stálo za to to udělat nekompatibilně a opravit to ve starých repech (nevím, jak často se to ve fykosu používá, já ify používám primárně pro ročenku a tolik jich zase není).

Optiony jsou fajn, zase se nedostává do kódu tolik balastu, ale otázka je, jak je práce s nimi příjemná (já třeba vím, jak dělat s těmi ify, ale jak s optiony, to je více advanced než základní jednoduché logické výrazy).

Reply:
Buď lze použít `\DeclareOption` ze samotného LaTeXu anebo `\opt` z balíku optional.
Tvrdím, že ify na semináře nechceme -- tyto lze zjistit z optionů.
U těch jazyků možná jo. Řešil bych to, až se bude dělat FoL, kde tohle vyplyne a bude jasnější, jaké use casy chceme. Howgh.

Reply:
Takže to mám přepsat nějak já jako rewiever nebo Lukáš? :D Jak se odvolávám na ty optiony? Declare ji deklaruji, ale jak v kódu poznám, který je aktuálně v platnosti?